integrations/jina/src/jina_haystack/__init__.py

```diff
@@ -0,0 +1,3 @@
+# SPDX-FileCopyrightText: 2023-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Please import the components here.
You can take [elasticsearch-haystack](https://github.com/deepset-ai/haystack-core-integrations/blob/f3ec5a015fa2d3ab5ef9a697fb75a23a3b58ae9f/integrations/elasticsearch/src/elasticsearch_haystack/__init__.py) as an example.